handlers/mxcontrol.js

```diff
@@ -0,0 +1,56 @@
+const nconf = require("nconf");
+const MXControlEvent = require("../events/mxcontrol");
+const Hipchat = require("../api/hipchat");
+const MXControl = require("mxcontrol");
+const Promise = require("bluebird");
+
+nconf.env("_");
+
+const hipchat = new Hipchat(nconf.get("HIPCHAT:ROOM:MXCONTROL:TOKEN"));
+const msgMXControlRoom =
+    (message) => hipchat.notify(message, {room: "MXControl"});
+
+const handleMXControlTask = (event) => {
+    const task = event.task;
+    const targetName = task.instance || task.environment || task.database;
+
+    const logline =
+        MXControl.buildLog(targetName, task.action, task.size, task.database);
+
+    msgMXControlRoom(logline);
+
+    return Promise.resolve(MXControl.runTask(task).then((res)=>{
```

Comment:
pls format better:
   - no parens around single param
   - spaces around `=>`
   - no empty line on function start

Reply:
The robut that was supposed to take care of it suffered from an insane attack of existential woe when I told it that that was its sole purpose for existing.

(i'll take care of it more completely when i get back